lib/stack_master/stack_definition.rb

```diff
@@ -32,6 +31,7 @@ def initialize(attributes = {})
       @ejson_file_kms = true
       @compiler = nil
       super
+      @additional_parameter_lookup_dirs ||= []
```

Comment:
`super` sets `@additional_parameter_lookup_dirs` to `nil` when not provided in the `stack_master.yml`. Moving this below, means we get a sensible default values of `[]`.